### ISSUE TITLE
Accessibility: Narrator doesn't announce collapsed/expanded input states on Chrome/Firefox 

### ISSUE THREAD

**Issue by sajsarka** [OfficeDev/office-ui-fabric-react#11381]
### Environment Information

- **Package version(s)**: Visio Data Visualizer Excel Addin
- **Browser and OS versions**: Windows 10 : Chrome - 78.0.3904.108 , Firefox - 70.0.1

### Describe the issue:

Narrator is not announcing collapsed/expanded state on Chrome and Firefox.

#### Actual behavior:

Narrator is not announcing collapsed/expanded state on Chrome and Firefox.

#### Expected behavior:

Screen reader should announced the state of the combobox and dropdown after getting expanded and collapsed.

### If applicable, please provide a codepen repro:

https://codepen.io/sajsarka/pen/gObpLRg

###Priorities and help requested
Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: Visio Data Visualizer Excel Addin

**Comment by xugao** [comment]
@sajsarka , please refer to this issue #11367 which has the same ask as yours.

Closing this one as it's duplicated. feel free to ask follow up questions there

**Comment by sajsarka** [comment]
I am able to reproduce the issue and the resolution mentioned in https://github.com/OfficeDev/office-ui-fabric-react/issues/11367 is not working for me.
Let me know if I am missing anything here.
Please use the code pen link to reproduce the issue : https://codepen.io/sajsarka/pen/gObpLRg
